senza/traffic.py

```diff
@@ -395,3 +396,12 @@ def inform_sns(arns: list, message: str, region):
     sns = boto3.client('sns', region_name=region)
     for sns_topic in sns_topics:
         sns.publish(TopicArn=sns_topic, Subject="SenzaTrafficRedirect", Message=jsonizer.encode((message)))
+
+
+def resolve_to_ip_addresses(dns_name: str) -> set:
+    try:
+        answers = dns.resolver.query(dns_name, 'A')
+    except:
```

Comment:
It really bothers me those "god mode" :godmode: exceptions catchers. Maybe we can try to be a bit more specific? Maybe [DNSException](https://github.com/rthalley/dnspython/blob/master/dns/exception.py#L19)?